docs/content/users/configuration/config.md

```diff
@@ -94,6 +94,14 @@ Composer version for the web container and the [`ddev composer`](../usage/comman
 | -- | -- | --
 | :octicons-file-directory-16: project | `2` | Can be `2`, `1`, or empty (`""`) for latest major version at container build time.<br><br>Can also be a minor version like `2.2` for the latest release of that branch, an explicit version like `1.0.22`, or a keyword like `stable`, `preview` or `snapshot`. See Composer documentation.
 
+Composer is updated during the first `ddev start` of a project, or when other changes like a new DDEV version require a rebuild of the web container. If it is important that every team member be on the exact same version of Composer, consider adding a post-start hook to your project such as:
+
+```yaml
+hooks:
+    post-start:
+        - exec: composer self-update 2.8.2
```

Comment:
The only situation I can think of where this `post-start` hook might be useful is when you have:

```
ddev config --composer-version "2"
```

Then decide to pin a version to:
```
ddev config --composer-version "2.8.2"
```
(Which builds a new image.)

And suddenly rollback to
```
ddev config --composer-version "2"
```
(and here Docker will not build a new image, it will use the cache from the beginning.)

But with this hook, Docker cache won't matter.

---

But this `post-start` hook has disadvantage of needing internet connection and downloading Composer on each `ddev start` (if the version inside differs).

---

What about a temporary pin
```
ddev config --composer-version "2.8.2" && ddev restart
```

And when a new DDEV release comes out, and you'll want to use the latest stable Composer again:
```
ddev config --ddev-version-constraint ">= v1.24.0" --composer-version "2" && ddev restart
```
The constraint will ensure that no Docker cache will be used, because you will be forced to update.